src/main/scala/com/snowplowanalytics/refererparser/scala/Parser.scala

```diff
@@ -39,6 +39,7 @@ object Medium extends Enumeration {
   val Internal = Value("internal")
   val Social   = Value("social")
   val Email    = Value("email")
+  val Paid     = Value("paid")
```

Comment:
Adding 'Paid' sources might be spun off in its own commit, which is independent from, and a prerequisite for, updating `referers.yml`. Up to you to decide if that would be 'cleaner'.

Reply:
I think one commit for this is enough.